streams/src/test/java/org/apache/kafka/streams/integration/MetricsIntegrationTest.java

```diff
@@ -310,8 +355,8 @@ private boolean testThreadMetric(final StringBuilder errorMessage) {
             testMetricByName(listMetricThread, TASK_CREATED_TOTAL, 1);
             testMetricByName(listMetricThread, TASK_CLOSED_RATE, 1);
             testMetricByName(listMetricThread, TASK_CLOSED_TOTAL, 1);
-            testMetricByName(listMetricThread, SKIPPED_RECORDS_RATE, 1);
-            testMetricByName(listMetricThread, SKIPPED_RECORDS_TOTAL, 1);
+            testMetricByName(listMetricThread, SKIPPED_RECORDS_RATE, 0);
```

Comment:
All the metrics with 0 expected values are left intentionally to validate that they are not created (either because only created lazily, or because they have been removed as proposed). Ditto elsewhere.